### PR TITLE
solution

### PR DESCRIPTION
**Pull request by venher-v**
https://venher-v.github.io/react_static-list-of-posts-js/

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,107 +1,35 @@
...
+}
 
-      <div className="PostInfo">
-        <div className="PostInfo__header">
-          <h3 className="PostInfo__title">
-            doloremque illum aliquid sunt
-          </h3>
+function getCommentById(postId) {
+  return commentsFromServer.filter(coment => coment.postId === postId)
+    || null;
```
Comment (anastasiiavorobiova) [inline src/App.jsx:15]:
```suggestion

```
This function never returns a null

File: src/components/PostInfo/PostInfo.jsx [hunk 2]
```diff
@@ -1,3 +1,46 @@
...
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        <p>
+          {' Posted by  '}
+          <UserInfo
```
Comment (anastasiiavorobiova) [inline src/components/PostInfo/PostInfo.jsx:21]:
```suggestion
          user && <UserInfo
```

Comment (anastasiiavorobiova) [reply src/components/PostInfo/PostInfo.jsx:21]: Consider adding a check if the user exists

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Almost done!

**Review by volodymyr-soltys97 — APPROVED** [review]
Great work 🔥

**Pull request closed without merging**